modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/client_serdes.ftl

```diff
@@ -291,13 +292,9 @@ public class ${schemaName}SerDes {
 						<#elseif allExternalSchemas?keys?seq_contains(propertyType) || allSchemas?keys?seq_contains(propertyType)>
 							${propertyType}SerDes.toDTO((String)jsonParserFieldValue)
 						<#elseif propertyType?ends_with("[]") && (allExternalSchemas?keys?seq_contains(propertyType?remove_ending("[]")) || allSchemas?keys?seq_contains(propertyType?remove_ending("[]")))>
-							Stream.of(
-								toStrings((Object[])jsonParserFieldValue)
-							).map(
-								object -> ${propertyType?remove_ending("[]")}SerDes.toDTO((String)object)
-							).toArray(
-								size -> new ${propertyType?remove_ending("[]")}[size]
-							)
+							TransformUtil.transform(toStrings((Object[])jsonParserFieldValue),
```

Comment:
@jeffwu0724 Please try to keep the format correct in ftls (although the generated classes will be formatted during regen).